test/app-tap/module_api.c

```diff
@@ -220,6 +228,8 @@ test_box_ibuf(lua_State *L)
 	fail_unless(ibuf_used(&ibuf) == 0);
 	fail_unless(*rpos == *wpos);
 
+	slab_arena_destroy(&arena);
+
```

Comment:
Nit: `slab_cache_destroy` missing.

Reply:
Doesn't the arena own the slab cache's memory?